### ISSUE TITLE
Unauthorized Error - region north_america

### ISSUE THREAD

**Issue by edwork** [bimmerconnected/bimmer_connected#233]
**Describe the issue**

The server returns a 401 UNAUTHORIZED error on connection. Discovered this via HASS first and the fingerprint command returns the same results. I've signed out and back into the app successfully via the same unchanged credentials. I drive a 2019 Mini Cooper FWIW. This happened around the time of the 0.117.2 release, possibly coincidentally. Last known working on Halloween (2020-10-31). 

**Expected behavior**

It returns a 200 or equivalent reponse

**Home Assistant version (if used)**

HASS 0.117.4

And did it work on a previous version, if so what was the last working version?

**Screenshots**

**ConnectedDrive website**

If you see any issues with the BMW integration in Home Assistant (or when directly using the library), please make sure to first login to the BMW ConnectedDrive website of your country and check if you can successfully login and query the car status.
- [x] I can still successfully login to the BMW ConnectedDrive website and the car status is available there.
- [x] I have ConnectedDrive enabled for my vehicle.

**Number of cars**

If you have more than 1 car please mention that here.
- [ ] I have 2 or more cars linked to the ConnectedDrive account.
- [x] I have a Mini vehicle linked to my account.

**Output of bimmer_connected fingerprint**

```
~/ ~/.local/bin/bimmerconnected fingerprint redacted@example.com redactedpass north_america
DEBUG:bimmer_connected.account:Getting vehicle list
DEBUG:bimmer_connected.account:getting new oauth token
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): customer.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://customer.bmwgroup.com:443 "POST /gcdm/usa/oauth/token HTTP/1.1" 200 None
DEBUG:bimmer_connected.account:got new token <REDACTED> with expiration date 2020-11-05 00:47:52.740450
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): b2vapi.bmwgroup.us:443
DEBUG:urllib3.connectionpool:https://b2vapi.bmwgroup.us:443 "GET /webapi/v1/user/vehicles HTTP/1.1" 401 0
DEBUG:bimmer_connected.account:The BMW Connected Drive portal returned an error: UNAUTHORIZED (received status code 401 and expected 200).
DEBUG:bimmer_connected.account:
Traceback (most recent call last):
  File "/home/edwork/.local/bin/bimmerconnected", line 10, in <module>
    sys.exit(main())
  File "/home/edwork/.local/lib/python3.7/site-packages/bimmer_connected/cli.py", line 240, in main
    args.func(args)
  File "/home/edwork/.local/lib/python3.7/site-packages/bimmer_connected/cli.py", line 116, in fingerprint
    log_responses=time_dir)
  File "/home/edwork/.local/lib/python3.7/site-packages/bimmer_connected/account.py", line 61, in __init__
    self._get_vehicles()
  File "/home/edwork/.local/lib/python3.7/site-packages/bimmer_connected/account.py", line 216, in _get_vehicles
    logfilename='vehicles')
  File "/home/edwork/.local/lib/python3.7/site-packages/bimmer_connected/account.py", line 152, in send_request
    raise IOError(msg)
OSError: The BMW Connected Drive portal returned an error: UNAUTHORIZED (received status code 401 and expected 200).
```

*NOTE: It might take us a while to respond on GitHub (we only maintain this in our free time), but we will respond.*

**Comment by rikroe** [comment]
Hey, while fixing the login for rest_of_world and china, we broke north_america.
Unfortunately, the APIs on BMW side seem a bit unstable and reverse engineering is a pain.

Downgrading to 0.117.2 should be a workaround for now and we'll see that it is adressed in the next update.

**Comment by rikroe** [comment]
Closed by #235.
